asciidoc-release-notes/adoc/attributes-product.adoc

```diff
@@ -20,12 +20,6 @@
 :z-product: {product} for IBM Z and LinuxONE
 :power-product: {product} for POWER
 
-:arm-product: {product} for {arm}
-:rpi-appliance: {arm-product} for the {rpi}
-:this-rpi-appliance: {arm-product} {this-version} for the {rpi}
-:z-product: {product} for IBM Z and LinuxONE
-:power-product: {product} for POWER
```

Comment:
Ugh, shoot, sorry for not noticing this in the previous PR.

Reply:
No problem, I didn't either.